src/hasReachedStyle.js

```diff
@@ -1,4 +1,6 @@
-export default function hasReachedStyle(currentStyle, style) {
+/* @flow */
+export default function hasReachedStyle(currentStyle: Object,
```

Comment:
I really think we need alias. It is not about Options being equal to Object, it is about idiomatic function API.

It also gives you ability to do compile-time checks if arguments are actually the same type. So if in one place you will use Options and in the other place Object - check will fail.

Often I have separate PropTypes file that exports something like:

```js
export const id = React.PropTypes.number;
export const name = React.PropTypes.string;
```